tsl/test/sql/continuous_aggs_ddl.sql

```diff
@@ -734,5 +734,31 @@ ALTER MATERIALIZED VIEW owner_check OWNER TO :ROLE_1;
 SELECT * FROM cagg_info WHERE user_view::text = 'owner_check';
 \x off
 
+--
+-- Test DROP SCHEMA CASCADE with continuous aggregates
+--
+-- Issue: #2350
+
+CREATE SCHEMA test_schema;
+
+CREATE TABLE test_schema.telemetry_raw (
+  ts        TIMESTAMP WITH TIME ZONE NOT NULL,
+  value     DOUBLE PRECISION
+);
+
+SELECT create_hypertable('test_schema.telemetry_raw', 'ts');
+
+CREATE MATERIALIZED VIEW test_schema.telemetry_1s
+  WITH (timescaledb.continuous)
+    AS
+SELECT time_bucket(INTERVAL '1 s', ts) AS ts_1s,
+       avg(value)
+  FROM test_schema.telemetry_raw
+ GROUP BY ts_1s WITH NO DATA;
+
+DROP SCHEMA test_schema CASCADE;
+
+SELECT true FROM pg_catalog.pg_namespace WHERE nspname = 'test_schema';
```

Comment:
Can you show that the internal objects (e.g., the materialized hypertable and views) are also removed from the internal namespaces (or whatever namespaces they existed in).